src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixMultipleCodeAction.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    /// <summary>
+    /// Fix multiple occurrences code action.
+    /// </summary>
+    internal partial class FixMultipleCodeAction : FixAllCodeAction
```

Comment:
this ordering seems reversed. I think it is more make sense to have

FixAllCodeAction : FixMultipleCodeAction

since FixAll is just one well known variation of FixMultiple I think.